Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,23 +1,18 @@
 import React from 'react';
-import './App.css';
+
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
+import Footer from './components/FooterComponents/Footer';
+
+import './App.scss';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+    <article class="social-card-holder">
```

Comment:
since the word 'class' is reserved in javascript, it is standard practice to use `className` when writing JSX code, this also has to do with the fact that JSX is working with DOM properties rather than HTML properties when it compiles. your class is still hitting the html element, but who knows what kinda weird bugs you might encounter in the future. past versions of react would refuse to render these unrecognized properties, but the latest version lets them through with a warning in your console. anyway, it is highly recommended to get into the habit of ALWAYS using className when writing JSX.

```<article className="social-card-holder">```